preferences/windows/xp/kustomization.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../base
```

Comment:
Should we just use `../base` here or does `hyperv` break things for XP?

https://github.com/kubevirt/common-instancetypes/blob/316b9e80649bfbe8cfe8034c61583581e2c5d393/preferences/windows/base/kustomization.yaml#L8-L18